src/remote/remote.go

```diff
@@ -683,6 +683,8 @@ func (c *Client) reallyExecute(tid int, target *core.BuildTarget, command *pb.Co
 		ActionDigest:    digest,
 		SkipCacheLookup: skipCacheLookup,
 	}, updateProgress)
+	log.Debug("completed ExecuteAndWaitProgress() for %v", target.Label)
```

Comment:
is this deliberate? or testing?

Reply:
I was going to open a PR separately but I think this might help diagnose some issues. Might be helpful to know when Please leaves that function to figure out where it's getting stuck "executing" or "queued".